databases/db.js

```diff
@@ -22,7 +23,9 @@ const Promotion = PromotionsModel(sequelize, Sequelize); //Nueva implementacion
 const game = gamesModel(sequelize, Sequelize);
 const team = teamsModel(sequelize, Sequelize);
 const inventory = inventoryModel(sequelize, Sequelize);
-const Deposit = 
+//const Deposit =
+const money= moneyModel(sequelize,Sequelize);
```

Comment:
Otra cosa!, esto NO ES solo dinero, es Dinero y Puntaje, pero mas que eso, esto es el registro de que un jugador esta participando en el fantasy y adicionalmente tiene el dinero y puntaje, yo le pondria
```suggestion
const PlayerFantasy = playerFantasyModel(sequelize,Sequelize);
```
O...
```suggestion
const FantasyParticipation = fantasyParticipationModel(sequelize,Sequelize);
```
O...
```suggestion
const PlayerFantasyParticipation = playerFantasyParticipationModel(sequelize,Sequelize);
```
O no se, ustedes lo ven